actionlist/header.js

```diff
@@ -41,28 +47,75 @@ const style = StyleSheet.create({
     marginRight: 20,
     justifyContent: 'flex-end',
   },
+  view: {
+    flex: 1,
+    backgroundColor: '#e9e9e9',
+    borderBottomWidth: 0.5,
+    paddingRight: 10,
+  },
+  text: {
+    color: '#7B1500',
+    margin: 10,
+    fontFamily: 'BentonSansBold, Arial, sans-serif',
+    fontWeight: 'bold',
+    fontSize: 15,
+    textAlign: 'left',
+    borderBottomWidth: 0.5,
+  },
 });
 
 /* eslint-disable global-require */
 /* rule disabled since image loading need not be global */
-const Header = () => (
-  <View style={style.toolbar}>
-    <Image
-      style={style.toolbarTrident}
-      source={require('./img/trident-large.png')}
-    />
-    <Text style={style.toolbarTitle}>
-      IU Action List
-    </Text>
-    <Image
-      style={style.toolbarSearch}
-      source={require('./img/search.png')}
-    />
-    <Image
-      style={style.toolbarButton}
-      source={require('./img/menu-icon.png')}
-    />
-  </View>
-  );
+export default class Header extends Component {
```

Comment:
I am using refs to open and close the drawer menu. I thought refs can't be used with stateless function. So i switched from it.